### PR TITLE
Release 1.6.0 (master)

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ topbar:
     title: Payments
   - url: /resources/
     title: Resources
+google_analytics:
```
Comment (asbjornu) [inline _config.yml]: We need to describe that Google Analytics is added in the release notes.

File: checkout/index.md [hunk 2]
```diff
@@ -124,7 +124,7 @@ sequenceDiagram
             Payer ->> Payer: Event: onPaymentCompleted
             Payer ->>+ Merchant: Check payment status
             deactivate Payer
-            Merchant ->>+ SwedbankPay: GET /psp/paymentorders/<paymentOrderId>
+            Merchant ->>+ SwedbankPay: GET <paymentorder.id>
```
Comment (NullableInt) [inline checkout/index.md]: Nice

File: resources/release-notes.md [hunk 3]
```diff
@@ -23,24 +23,37 @@ sidebar:
...
 
+### Version 1.6
+
+* [Credit Payments][credit] (née "Credit Account") sections have been deleted
+  from the Swedbank Pay Developer Portal.
+* The [Swish Payments][swish] and [Vipps Payments][vipps] have received a few
+  updates and are now ready for another round of review.
+* [MobilePay Payments][mobile-pay] is now ready for review.
+* `paymentAgeLimit` is now added in [Swish Payments][swish].
+* The Terminology page][terminology] is updated and *3-D Secure 2.0 (3DS2)* ,
```
Comment (asbjornu) [inline resources/release-notes.md]:
```suggestion
* The [Terminology page][terminology] is updated and *3-D Secure 2.0 (3DS2)* ,
```

File: resources/release-notes.md [hunk 4]
```diff
@@ -23,24 +23,37 @@ sidebar:
...
+  *PSD2* and *SCA* explanations are added.
+* Added description for Payment States And Transaction States.
```
Comment (asbjornu) [inline resources/release-notes.md]:
```suggestion
* Added descriptions for Payment States And Transaction States everywhere appropriate.
```

**Pull request merged**